UI/dist/obs.desktop

```diff
@@ -2,7 +2,9 @@
 Version=1.0
 Name=OBS
 GenericName=Streaming/Recording Software
+GenericName[fr]=Logiciel d'enregistrement/diffusion
 Comment=Free and Open Source Streaming/Recording Software
+Comment[fr]=Logiciel d'enregistrement et de diffusion gratuit et à code source ouvert
```

Comment:
Would be best with `sous licence libre` instead of `à code source ouvert`.

Reply:
Also, `free` in FOSS means `free as in freedom`, not `free as in free beer`.

Reply:
The EN -> FR translation of "free" isn't always the same amongst different projects, hence I choose to stay literal/conservative with this translation.

If the original meaning here is "free as in beer and in freedom", then "sous licence libre" is fine and the "gratuit" can be removed since it would duplicates.

Given the project states that the licence is GPL 2+, I'm changing it this way.